src/types/fetch.ts

```diff
@@ -93,15 +93,4 @@ export interface $Fetch<
   ): $Fetch<T, R>;
 }
 
-declare global {
```

Comment:
Can we please still keep this? I like to make types working as much as possible before prepare command.

Reply:
I am worried that the types may conflict but I'm happy to restore it and see.